### PR TITLE
Trestle create command

### PR DESCRIPTION
**Pull request by butler54**
## Types of changes

- \[ \] Bug fix (non-breaking change which fixes an issue)
- \[x\] New feature (non-breaking change which adds functionality)
- \[ \] Breaking change (fix or feature that would cause existing functionality to change)
- \[x\] My code follows the code style of this project.
- \[x\] My change requires a change to the documentation.
- \[x\] I have updated the documentation accordingly.
- \[x\] I have added tests to cover my changes.
- \[x\] All new and existing tests passed.
- \[x\] All commits are signed.
 
Completed first pass implementation of trestle create. Additional functionality (like default splits) could be added, however, have not been yet.

### CODE REVIEW

**Review by aNebula — COMMENTED** [review]
LGTM

**Review by aNebula — APPROVED** [review]
LGTM

File: trestle/core/generators.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    """
+    # FIXME: Should be in separate generator module as it inherits EVERYTHING
+    if type_ is datetime:
+        return datetime.now().astimezone()
+    elif type_ is bool:
+        return False
+    elif type_ is int:
+        return 0
+    elif type_ is str:
+        return 'REPLACE_ME'
```
Comment (fsuits) [inline trestle/core/generators.py:51]: There is a python faker module that looks ok - and it makes uuids.

**Review by fsuits — APPROVED** [review]
Looks good - I just had some comments.  I will use it as an example so let me know if anything is not intended as a forward looking approach

**Pull request merged**